### PR TITLE
Rename PAT to personal access token, move to separate traits, files and directories

### PR DESCRIPTION
**Pull request by spetz**

### CODE REVIEW

File: iggy/src/binary/users.rs [hunk 1]
```diff
@@ -2,24 +2,17 @@ use crate::binary::binary_client::BinaryClient;
 use crate::binary::mapper;
 use crate::bytes_serializable::BytesSerializable;
 use crate::command::{
-    CHANGE_PASSWORD_CODE, CREATE_PERSONAL_ACCESS_TOKEN_CODE, CREATE_USER_CODE,
-    DELETE_PERSONAL_ACCESS_TOKEN_CODE, DELETE_USER_CODE, GET_PERSONAL_ACCESS_TOKENS_CODE,
-    GET_USERS_CODE, GET_USER_CODE, LOGIN_USER_CODE, LOGIN_WITH_PERSONAL_ACCESS_TOKEN_CODE,
-    LOGOUT_USER_CODE, UPDATE_PERMISSIONS_CODE, UPDATE_USER_CODE,
+    CHANGE_PASSWORD_CODE, CREATE_USER_CODE, DELETE_USER_CODE, GET_USERS_CODE, GET_USER_CODE,
+    LOGIN_USER_CODE, LOGOUT_USER_CODE, UPDATE_PERMISSIONS_CODE, UPDATE_USER_CODE,
```
Comment (hubcio) [inline iggy/src/binary/users.rs]: use crate::command::*;

File: iggy/src/binary/personal_access_tokens.rs [hunk 2]
```diff
@@ -0,0 +1,54 @@
+use crate::binary::binary_client::BinaryClient;
+use crate::binary::mapper;
+use crate::bytes_serializable::BytesSerializable;
+use crate::command::{
```
Comment (hubcio) [inline iggy/src/binary/personal_access_tokens.rs]: use crate::command::*;

File: server/tests/streaming_tests/personal_access_token.rs [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    let (pat1, raw_token1) = PersonalAccessToken::new(1, "test1", now, None);
+    let (pat2, raw_token2) = PersonalAccessToken::new(2, "test2", now, Some(1000));
+    let (pat3, raw_token3) = PersonalAccessToken::new(3, "test3", now, Some(100_000));
+
+    setup
+        .storage
+        .personal_access_token
+        .save(&pat1)
+        .await
+        .unwrap();
```
Comment (hubcio) [inline server/tests/streaming_tests/personal_access_token.rs]: why unwraps? pls use expect() with proper string

File: server/src/streaming/systems/personal_access_tokens.rs [hunk 4]
```diff
@@ -0,0 +1,123 @@
+use crate::streaming::personal_access_tokens::personal_access_token::PersonalAccessToken;
+use crate::streaming::session::Session;
+use crate::streaming::systems::system::System;
+use crate::streaming::users::user::User;
+use iggy::error::Error;
+use iggy::utils::text;
+use iggy::utils::timestamp::TimeStamp;
+use tracing::{error, info};
+
+const MAX_PERSONAL_ACCESS_TOKENS: u32 = 100;
```
Comment (hubcio) [inline server/src/streaming/systems/personal_access_tokens.rs:10]: for future: make it configurable

File: iggy/src/binary/users.rs (hunk 1 above)
Comment (spetz) [reply iggy/src/binary/users.rs]: Fixed

File: iggy/src/binary/personal_access_tokens.rs (hunk 2 above)
Comment (spetz) [reply iggy/src/binary/personal_access_tokens.rs]: Fixed

File: server/tests/streaming_tests/personal_access_token.rs (hunk 3 above)
Comment (spetz) [reply server/tests/streaming_tests/personal_access_token.rs]: Fixed

File: server/src/streaming/systems/personal_access_tokens.rs (hunk 4 above)
Comment (spetz) [reply server/src/streaming/systems/personal_access_tokens.rs:10]: Good idea :)

**Review by hubcio — APPROVED** [review]

**Pull request merged**